### PR TITLE
[master] fix(middlewared/vm): Fix vnc bind interface issue. Now bhyves call vnc_init_server passing IPV4 address attached on fbuf.

### PR DESCRIPTION
**Pull request by araujobsd**
Note: It depends of libhyve-remote 0.1.4.1 and also os/pull/53.

Ticket: #24884

### CODE REVIEW

File: src/middlewared/middlewared/plugins/vm.py [hunk 1]
```diff
@@ -563,6 +563,14 @@ def identify_hypervisor(self):
                 vnc_devices.append(vnc)
         return vnc_devices
 
+    @accepts()
+    def get_vnc_ifaces(self):
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/vm.py]: If I were a user and saw that in /api/docs I would expect `vm.get_vnc_ifaces` to return an interface list: `["em0", "igb0"]`.

Would you agree its more obvious to use `get_vnc_ips` ?

**Comment by william-gr** [comment]
Jenkins: retest this please

**Comment by kmoore134** [comment]
Jenkins: retest this please

**Comment by araujobsd** [comment]
Jenkins: retest this please

**Comment by kmoore134** [comment]
Jenkins: retest this please

**Pull request merged**